### DISCUSSION TITLE
Why does my Honor shrink?

### DISCUSSION THREAD

**Discussion by Eisi05** [codewars/codewars.com#2933] [Q&A] [answered]
![Screenshot (768)](https://user-images.githubusercontent.com/74506494/217083610-2d51724f-5713-4a95-828d-858dfd460cc6.png)
So, I just got over 1000 honor and also the rank 3 kyu in Java, but when i loaded the page like 10min later I have only 650 honor.
Is there any reason behind it?
Thanks for help

**Comment by Blind4Basics** [comment, marked as answer]
Hello,

Yes, "stealing" code on the internet is not considered a valid way to solve kata. So your last completions got invalidated.

Cheers
